.github/workflows/ci.yml

```diff
@@ -9,28 +9,59 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    
+
+    services:
```

Comment:
If you have it like this, these services will be running even for builds that don't need them like the Spark build, the Cassandra builds etc... that's an issue because these images are running directly in the build containers (i.e. where build.sh is running) and are taking memory/disk from them (which is already very close to the limit). I was hoping that these images are actually run in separate containers from the build containers so adding them here would save memory/disk but that is not the case. I asked on the github-actions threads have a look here: https://github.community/t/where-are-services-hosted/178281


In the ProtoQuill build I changed the setup_databases script to bring up the postgres/mysql images that we already have in our docker-compose. Maybe we want to do it that way instead?

Have a look at my modified `setup_databases.sh` here: https://github.com/getquill/protoquill/blob/1223e50b17dcc9a555f9599860493bb2c439351f/build/setup_databases.sh#L8
I also had to change `setup_db_scripts.sh` too:
https://github.com/getquill/protoquill/blob/master/build/setup_db_scripts.sh